source/_posts/2017-08-29-gsoc-2017-support-to-import-export-of-more-formats.markdown

```diff
@@ -0,0 +1,274 @@
+---
+layout: post
+title: "GSoC 2017 : Support to Import & Export of more formats"
+date: 2017-08-29 17:16
+comments: true
+author: Athitya Kumar
+categories: [GSOC 2017, GSOC, Daru, Daru-IO, Import, Export]
+---
+
+## Introduction
+
+> "Hello friend. Hello friend? That's lame." - S01E01 (Pilot), Mr.Robot
+
+My name is Athitya Kumar, and I'm a 4th year undergrad from IIT Kharagpur, India. I
+was selected as a GSoC 2017 student developer by Ruby Science Foundation for project daru-io.
+
+Daru-IO is a plugin-gem to daru gem, that extends support for many Import and Export methods
+of `Daru::DataFrame`. This gem is intended to help Rubyists who are into Data Analysis
+or Web Development, by serving as a general purpose conversion library.
+
+Through this summer, I worked on adding support for various Importers and Exporters
+while also porting some existing modules. Please feel free to find a comprehensive set
+of useful links in 
+[Final Work Submission](https://athityakumar.github.io/blog/posts/GSoC_2017_-_Final_Work_Submission/)
+and [README](https://github.com/athityakumar/daru-io/blob/master/README.md). Before proceeding any
+further, the reader might also be interested in having a look at
+[Rails example](https://daru-examples-io-view-rails.herokuapp.com/) and
+[the code](https://github.com/Shekharrajak/daru_examples_io_view_rails) making it work.
+
+## Mark Anthony's Speech (ft. daru)
+
+> "Rubyists, Data Analysts and Web Developers, lend me your ears;
+
+> I come to write about my GSoC project, not to earn praise for it."
```

Comment:
Do this:
```
> "Rubyists, Data Analysts and Web Developers, lend me your ears;
>
> I come to write about my GSoC project, not to earn praise for it."
```

It will look like this:
> "Rubyists, Data Analysts and Web Developers, lend me your ears;
>
> I come to write about my GSoC project, not to earn praise for it."

Instead of this:
> "Rubyists, Data Analysts and Web Developers, lend me your ears;

> I come to write about my GSoC project, not to earn praise for it."

(Unless Octopress' Markdown engine processes it differently)

Reply:
Indeed, Octopress' markdown processes it differently. :smile:

![screen shot 2017-09-01 at 7 29 41 pm](https://user-images.githubusercontent.com/17109060/29973068-dadd8990-8f4b-11e7-8a57-95d0b840eced.png)